src/jtag3.c

```diff
@@ -2334,7 +2261,7 @@ int jtag3_read_sib(const PROGRAMMER *pgm, const AVRPART *p, char *sib) {
 
   memcpy(sib, resp+3, AVR_SIBLEN);
   sib[AVR_SIBLEN] = 0; // Zero terminate string
-  avrdude_message(MSG_DEBUG, "%s: jtag3_read_sib(): Received SIB: \"%s\"\n", progname, sib);
+  pmsg_debug("jtag3_read_sib(): received SIB: %s\n", sib);
```

Comment:
@stefanrueger there are a few places where the hardcoded function name is still present in the string, even though you're using `pmsg_level`. Is it simply a typo/error, or is it necessary?

Reply:
Thanks for reviewing. Ahh, the function names are *only* automatically added at warning and error levels (and only when `-v` is given). They are *not* added at debug levels (or any other), so they need to stay in the message as the author thought that is useful for their debugging.

Reply:
As background, here how the pmsg_level() functions are defined and where you can see that only error and warning print the function name:

https://github.com/avrdudes/avrdude/blob/e20b9e8e585cb07e7bde685d35250d3ae99715a1/src/avrdude.h#L69-L77